pyproject.toml

```diff
@@ -11,6 +11,7 @@ packages = [{ include = "compute_modules" }]
 
 [tool.poetry.dependencies]
 python = "^3.9"
+requests = "^2.32.3"
```

Comment:
....mhhh, could we redo this pr without dependencies. The goal of the base image is to be dependency free